packages/web/src/js/Collapse.ts

```diff
@@ -35,17 +37,17 @@ class Collapse extends BaseComponent {
 
   constructor(element: HTMLElement) {
     super(element);
-    this.target = this.element.dataset.target ? selectorEngine.findOne(`#${this.element.dataset.target}`) : null;
+    this.target = this.element.dataset.target ? SelectorEngine.findOne(`#${this.element.dataset.target}`) : null;
```

Comment:
This is the same as https://github.com/lmc-eu/spirit-design-system/pull/583#discussion_r1032439263

Reply:
Already solved this